pyairtable/api/table.py

```diff
@@ -150,6 +150,28 @@ def batch_delete(self, record_ids: List[str]):
         but without ``base_id`` and ``table_name`` arg.
         """
         return super()._batch_delete(self.base_id, self.table_name, record_ids)
+    
+    def get_comments(self, record_id: str):
+        meta_response = super()._get_comments(self.base_id, self.table_name, record_id)
+        comments = meta_response.get('comments')
+        offset = meta_response.get('offset')
+        while offset is not None:
+            meta_response = super()._get_comments(self.base_id, self.table_name, record_id, offset=offset)
+            comments += meta_response.get('comments')
+            offset = meta_response.get('offset')
+        return comments
+    
+    def get_comments_url(self, record_id: str, comment_id: str = ''):
```

Comment:
why is comment_id optional? Is method intended to be used for both single fetch + list fetch?

Reply:
I originally just added that in for the tests, but I've gone and removed it now. There is no single get comment method, it was referring to getting the URL for comments, as the request methods get/patch/post/delete are used rather than separate resources.
The url's are built in each method for getting comments, rather than a separate method.